CVEs/scripts/scan_vuln.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+images_with_tags=$(cat images.txt)
+
+echo "" > scan_results.txt
+
+for line in $images_with_tags; do
+  trivy image --platform=linux/amd64  --no-progress --output scan.tmp --format json --severity CRITICAL "$line"
+  if [ $? -ne 0 ]; then
+    echo "$line | ERROR PROCESSING! " >> scan_results.txt
```

Comment:
this will break the markdown table formatting because of the missing columns.

also, I think we should we `rm` the `scan.tmp` file here to have a clean state in the next iteration of the loop. Maybe it would be useful to print the received error somewhere.